crates/ironrdp/examples/server.rs

```diff
@@ -389,10 +332,10 @@ async fn run(
 
     let server_builder = if let Some((cert_path, key_path)) = cert.as_deref().zip(key.as_deref()) {
         let identity = TlsIdentityCtx::init_from_paths(cert_path, key_path).context("failed to init TLS identity")?;
-        let acceptor = acceptor(identity.cert, identity.priv_key).context("failed to build TLS acceptor")?;
+        let acceptor = identity.make_acceptor().context("failed to build TLS acceptor")?;
 
         if hybrid {
-            server_builder.with_hybrid(acceptor, identity.pub_key)
+            server_builder.with_hybrid(acceptor, identity.pub_key().into())
```

Comment:
If the previous suggestion is applied, we can revert this to
```suggestion
            server_builder.with_hybrid(acceptor, identity.pub_key)
```